internal/guest/runtime/hcsv2/uvm.go

```diff
@@ -648,6 +718,12 @@ func (h *Host) ExecProcess(ctx context.Context, containerID string, params prot.
 	var pid int
 	var c *Container
 	var envToKeep securitypolicy.EnvList
+
+	userName, groupNames, umask, err := getUserInfo(params.OCISpecification)
```

Comment:
some work is already done in `setupWorkloadContainerSpec`. Do we need to be re-parsing things again here?

Reply:
As discussed, changes in containerd make this less of a concern.